com.wso2telco.dep.gw.ussdapi/ussdapigw-synapse/src/main/synapse-config/sequences/sendUssdSeq.xml

```diff
@@ -11,14 +11,13 @@
     </filter>
     <filter source="boolean(get-property('originalNotifyURL'))" regex="false">
         <then>
-            <class name="org.wso2telco.dep.nashornmediator.NashornMediator">
-                <property name="script" value='
-                var payload= mc.getPayloadJSON();
-                var str = mc.getProperty("outboundUSSDMessage");
-                var text = str.replace("\n", "\\n").replace("\r", "\\r");
-                payload.outboundUSSDMessageRequest.outboundUSSDMessage = text;
-                mc.setPayloadJSON(payload);
-                '/>
+            <property expression="fn:replace(fn:replace(syn:get-property('outboundUSSDMessage'), '\n', '\\n'), '\r', '\\r')" name="propertyReplacement" scope="default" type="STRING" xmlns:fn="http://www.w3.org/2005/xpath-functions"/>
```

Comment:
It seems this block of code has been duplicated in both sendUssdSeq.xml and modifySendUSSDRequestPayloadSeq sequences. I think this can be implemented without considering the value of "subsEndPoint" property. Could you please check again?